explorer/explorertypes.go

```diff
@@ -43,24 +43,29 @@ type TxBasic struct {
 
 // AddressTx models data for transactions on the address page
 type AddressTx struct {
-	TxID          string
-	InOutID       uint32
-	Size          uint32
-	FormattedSize string
-	Total         float64
-	Confirmations uint64
-	Time          int64
-	FormattedTime string
-	ReceivedTotal float64
-	SentTotal     float64
-	IsFunding     bool
-	MatchedTx     string
-	BlockTime     uint64
+	TxID           string
+	InOutID        uint32
+	Size           uint32
+	FormattedSize  string
+	Total          float64
+	Confirmations  uint64
+	Time           int64
+	FormattedTime  string
+	ReceivedTotal  float64
+	SentTotal      float64
+	IsFunding      bool
+	MatchedTx      string
+	BlockTime      uint64
+	MergedTxnCount uint64 `json:"MergedTxnCount,omitempty"`
```

Comment:
I only tagged it because  I did not want it to be part of the payload sent to the frontend when `merged debit view` address history page is not being viewed.

Reply:
I will tag the rest to ensure consistency.

Reply:
Actually this is all i need to set``json:",omitempty"`` there.